public/electron.js

```diff
@@ -56,6 +61,23 @@ const windowStateKeeper = windowName => {
 };
 
 const createWindow = async () => {
+  // Install chrome extensions for React and Redux for the Electron windows
```

Comment:
Are these extensions installed on a "normal" users, i.e. when running the binaries? If they are, maybe that something we should avoid? With some `isDevMode()` check or something,

Reply:
Good catch! Yeah, they are installed even when not in dev-mode.
I'll update it :)